### PR TITLE
DOCS-1076: Simplify MR landing pages

### PR DESCRIPTION
**Pull request by andf-viam**
First stab at refactoring MR landing pages.

- Initial dumping of remotes and subtypes into an `advanced` left-nav entry. This likely not ideal, but good to refocus on rest of MR stuff as we start considering options.
- This mostly just for team brainstorming!
- Caught many aliases, but probably have a few more to catch.

### CODE REVIEW

File: docs/modular-resources/key-concepts.md [hunk 1]
```diff
@@ -16,8 +16,15 @@ description: "The key concepts behind how Viam's resource APIs and models are un
 no_list: true
 ---
 
-You can extend the features of an existing {{< glossary_tooltip term_id="resource" text="resource" >}} API by [creating a custom module](/extend/modular-resources/create/) to define a new [model](#models) that implements that API.
-A custom module can provide one or more modular resource models.
+Viam's [Robot Development Kit (RDK)](/internals/rdk/) provides built-in support for a variety of {{< glossary_tooltip term_id="resource" text="resources" >}}:
```
Comment (npentrel) [inline docs/modular-resources/key-concepts.md:19]: I wonder if these few lines are more approachable than what’s on the top level page now but we can address that separately

**Review by npentrel — CHANGES_REQUESTED** [review]
I think we are missing some aliases. Every page currently under extend needs an alias to where it was. Otherwise good!

File: docs/modular-resources/_index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Extend Viam with Modular Resources"
 linkTitle: "Modular Resources"
-weight: 10
+weight: 50
```
[the remark this answers is not in the document]
Comment (andf-viam) [reply docs/modular-resources/_index.md:4]: I've fixed this in [most recent commit](https://github.com/viamrobotics/docs/pull/1998/commits/bdc3101ebe20bf90af856ff742edfeaab1910b0b).

File: docs/modular-resources/key-concepts.md (hunk 1 above)
Comment (andf-viam) [reply docs/modular-resources/key-concepts.md:19]: Let's try it!

**Comment by andf-viam** [comment]
@npentrel Thanks, you're absolutely right. Think I got all the missing aliases now!

File: docs/modular-resources/create/_index.md [hunk 3]
```diff
@@ -14,21 +14,23 @@ tags:
   ]
 description: "Use the Viam module system to implement modular resources that can be included in any Viam-powered smart machine."
 no_list: true
+aliases:
+  - "/extend/modular-resources/create/create-subtype/"
```
Comment (npentrel) [inline docs/modular-resources/create/_index.md:18]: the create-subtype page is covered in that file
```suggestion
  - "/extend/modular-resources/create/"
```

**Review by npentrel — APPROVED** [review]
One remaining alias to fix

**Pull request merged**